index.html

```diff
@@ -0,0 +1,634 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contributing to Open Source Part I: The Easy Way - YouTube</title>
+    <link rel="icon" href="image/youtube.png" type="image/png">
+    <meta name="author" content="C. Miguel V. Vega">
+    <meta name="description" content="Embedding Images and Video.">
+    <meta name="keywords" content="HTML, CSS">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="flex-container">
+            <div class="nav-icon">
+                <button class="material-icons">menu</button>
+            </div>
+
+            <div class="navbar-logo">
+                <img src="image/yt_logo_rgb_light.png"
+                alt="Logo Youtube">
+            </div>
+
+            <div id="search">
+                <form action="https://www.google.com/">
+                    <div class="container-form">
+                        <div style="width:100%">
+                            <div class="container-form2">
+                                <div id="container-icon">
+                                    <div class="container-icon2">
+                                        <a href="">
+                                            <span class="material-icons">keyboard</span>
+                                        </a>
+                                    </div>
+                                    <div class="flex-container">
+                                        <input type="text" value="" placeholder="Search">
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <button type="submit" class="material-icons">search</button>
+                </form>
+            </div>
+
+            <div id="buttons">
+                <div class="flex-container">
+                    <button class="material-icons">video_call</button>
+                </div>
+                <div class="flex-container">
+                    <button class="material-icons">apps</button>
+                </div>
+                <div class="flex-container">
+                    <button class="material-icons">sms</button>
+                </div>
+                <div class="flex-container">
+                    <button class="material-icons">notifications</button>
+                </div>
+                <div class="flex-container">
+                    <img id="image-profile" src="image/photo.jpg" alt="photo of user">
```

Comment:
`SUGGESTION`

Consider adding the images you referenced. You seem to be adding images that are not available on the repo.